SECURITY.md

```diff
@@ -0,0 +1,24 @@
+# Security Policy
```

Comment:
What do we think are the security concerns with `pydagmc`? We should what they are here, but leave room for unexpected concerns too. Realistically, I don't foresee any security (e.g. personal data, etc.) concerns with this tool.

Reply:
Yeah let's pull it unless there's meaningful content or concerns to address.